examples/jsm/loaders/GLTFLoader.js

```diff
@@ -574,13 +574,11 @@ class GLTFLightsExtension {
 				break;
 
 			case 'point':
-				lightNode = new PointLight( color );
-				lightNode.distance = range;
+				lightNode = new PointLight( color, 1, range, 2 );
 				break;
 
 			case 'spot':
-				lightNode = new SpotLight( color );
-				lightNode.distance = range;
+				lightNode = new SpotLight( color, 1, range, 2 );
```

Comment:
The fourth parameter of `SpotLight` is `angle`, not `decay`. 

I suggest you don't change the ctor line and just add `lightNode.decay = 2;` for spot and point lights.

Reply:
Um, not sure the line is needed at all since the default value of `decay` is now always 2.